scripts/stripe/fix-stale-new-paymentIntents.ts

```diff
@@ -0,0 +1,54 @@
+import '../../server/env';
+
+import moment from 'moment';
+
+import OrderStatuses from '../../server/constants/order-status';
+import { PAYMENT_METHOD_SERVICE, PAYMENT_METHOD_TYPE } from '../../server/constants/paymentMethods';
+import { syncOrder } from '../../server/lib/stripe/sync-order';
+import { Collective, Op, Order, PaymentMethod } from '../../server/models';
+
+const main = async () => {
+  const results = await Order.findAndCountAll({
+    limit: 20,
```

Comment:
We should either remove/increase the limit or implement pagination:
```suggestion
```

Reply:
Added pager! Thanks